app/views/layouts/_lte_sidebar.html.erb

```diff
@@ -170,52 +170,97 @@
       </li>
     </ul>
   </li>
-  <% if current_user.has_role?(Role::ORG_ADMIN, current_organization) %>
-    <li class="nav-item has-treeview <%= menu_open?(['audits', 'reports']) %>">
-      <a href="#" class="nav-link <%= active_class(['audits', 'reports']) %>">
-        <i class="nav-icon fa fa-tasks"></i>
-        <p>Reporting & Auditing
-          <i class="fas fa-angle-left right"></i>
-        </p>
-      </a>
-      <ul class="nav nav-treeview">
+  <li class='nav-item has-treeview <%= menu_open?(['audits', 'reports/annual_reports', 'distributions_by_county', 'reports/itemized_donations', 'reports/itemized_distributions', 'reports/donations_summary', 'reports/activity_graph', 'reports/manufacturer_donations_summary', 'reports/product_drives_summary', 'purchases_summary', 'distributions_summary']) %>'>
+    <a href='#' class='nav-link <%= active_class(['audits', 'reports/annual_reports', 'distributions_by_county', 'reports/itemized_donations', 'reports/itemized_distributions', 'reports/donations_summary', 'reports/activity_graph', 'reports/manufacturer_donations_summary', 'reports/product_drives_summary', 'purchases_summary', 'distributions_summary']) %>'>
+      <i class="nav-icon fa fa-tasks"></i>
+      <p>Reporting & Auditing
```

Comment:
This moves the reporting menu so it's visible by anyone, not just org admins. Was that on purpose?

Reply:
Yes -- the sub-menu of `Inventory Audit` is now the only one protected, on purpose.